lib/scrabble.rb

```diff
@@ -1,7 +1,36 @@
 class Scrabble
 
+  def standardized_format(word)
+    word.upcase
+  end
+
   def score(word)
-    1
+    return 0 if word.nil?
+
+    values = point_values
+    word = standardized_format(word)
+
+    sum = 0
+    word.each_char do |char|
+      sum += values[char]
+    end
+    sum
+  end
+
+  def score_with_multipliers(word, board_array, multiplier = 1)
```

Comment:
Think about ways that you could break the logic in this method into different parts (new methods).